.github/workflows/ci.yml

```diff
@@ -50,14 +50,12 @@ jobs:
         # See https://github.com/actions/runner-images/issues/183
         #
         # For Arch Linux uml is not yet supported, so only test under qemu there.
-        os: [bookworm]
-        backend: [qemu, uml, kvm]
```

Comment:
Looks like we now do qemu  & kvm on all os', so i guess we can simplify te below again by using the matrix rather then including everything by hand?